arcgis-ios-sdk-samples/Maps/Generate offline map (overrides)/GenerateOfflineMapOverrides.storyboard

```diff
@@ -52,11 +51,11 @@
                                     </label>
                                     <progressView opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="750" progress="0.5" translatesAutoresizingMaskIntoConstraints="NO" id="GNP-kU-lcZ">
                                         <rect key="frame" x="8" y="33.5" width="303" height="2"/>
-                                        <color key="tintColor" red="0.0" green="0.36862745099999999" blue="0.58431372550000005" alpha="1" colorSpace="calibratedRGB"/>
+                                        <color key="tintColor" name="primaryThemeColor"/>
```

Comment:
Could you please set the global tint color of the storyboard rather than the tint color on individual views?